CHANGELOG.md

```diff
@@ -305,7 +306,7 @@ It is possible to migrate critical data from the old db to the sqlite but there
 - Fixed an issue where Alice would not verify if Bob's Bitcoin lock transaction is semantically correct, i.e. pays the agreed upon amount to an output owned by both of them.
   Fixing this required a **breaking change** on the network layer and hence old versions are not compatible with this version.
 
-[Unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.10.2...HEAD
+[unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.10.2...HEAD
```

Comment:
Is this change on purpose? Was the change triggered by formatter? 🤔

Reply:
d print changed this